src/device_base.cc

```diff
@@ -27,6 +27,7 @@ void fastcat::DeviceBase::SetLoopPeriod(double loop_period)
 
 void fastcat::DeviceBase::SetTime(double time, double monotonic_time)
 {
+  last_time_             = state_->monotonic_time;
```

Comment:
you need to differentiate between monotonic and standard time, they are two different clock sources..

can you change this variable to `last_monotonic_time_` ?

Reply:
Done!